main.go

```diff
@@ -13,7 +13,7 @@ import (
 var version = "0.0.0"
 
 func main() {
-	ghTkn := os.Getenv("GITHUB_TOKEN")
+	ghTkn := os.Getenv("GITHUB_ACCESS_TOKEN")
 	if ghTkn == "" {
 		log.Fatalf("'GITHUB_TOKEN' env var required")
```

Comment:
@vdice yup, renamed in https://github.com/deis/deisrel/pull/48/commits/431151cbcdde3906ac02283d6bc70a4e9cf72780